src/_data/glossary.yml

```diff
@@ -1,3 +1,17 @@
+- term: "Assist"
+  short_description: |-
+    An automated, local code edit targeted at common improvements and conventions
+    based on the structure of the code.
+  related_links:
```

Comment:
What do you think about adding a related link to the "Refactor" entry, since the terms are both code edits that don't require a diagnostic. Then one back to assist for refactor.